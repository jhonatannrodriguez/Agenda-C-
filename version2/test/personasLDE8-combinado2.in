crearListaPersonas

crearAgenda
crearPersona 1 18 Persona1
crearFecha 18/4/2023
crearEvento 1 Descripcion uno
agregarEventoPersona
insertarPersonaListaPersonas 1

crearAgenda
crearPersona 2 20 Persona 2
insertarPersonaListaPersonas 1

crearAgenda
crearPersona 3 25 Persona3
insertarPersonaListaPersonas 2

crearAgenda
crearPersona 4 30 Persona4
crearFecha 19/4/2023
crearEvento 1 Descripcion uno
agregarEventoPersona
crearFecha 20/4/2023
crearEvento 2 Descripcion dos
agregarEventoPersona
insertarPersonaListaPersonas 4

imprimirCantidadListaPersonas
imprimirListaPersonas

estaEnListaPersonas 5
estaEnListaPersonas 2

eliminarInicioListaPersonas

estaEnListaPersonas 2

imprimirListaPersonas

concatenarConVacioListaPersonas

imprimirListaPersonas

imprimirPersonaListaPersonas 3
imprimirPersonaListaPersonas 4

liberarListaPersonas

Fin
