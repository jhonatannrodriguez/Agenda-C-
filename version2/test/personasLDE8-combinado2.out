1>2>3>4>5>6>7>8>9>10>11>12>13>14>15>16>17>18>19>20>21>22>23>La cantidad de personas en la lista es: 4.
24>Persona 2:  Persona 2, 20 años
Persona 3:  Persona3, 25 años
Persona 1:  Persona1, 18 años
Evento 1:  Descripcion uno
Fecha: 18/4/2023
Persona 4:  Persona4, 30 años
Evento 1:  Descripcion uno
Fecha: 19/4/2023
Evento 2:  Descripcion dos
Fecha: 20/4/2023
25>La persona con id 5 NO está en la lista.
26>La persona con id 2 está en la lista.
27>28>La persona con id 2 NO está en la lista.
29>Persona 3:  Persona3, 25 años
Persona 1:  Persona1, 18 años
Evento 1:  Descripcion uno
Fecha: 18/4/2023
Persona 4:  Persona4, 30 años
Evento 1:  Descripcion uno
Fecha: 19/4/2023
Evento 2:  Descripcion dos
Fecha: 20/4/2023
30>31>Persona 3:  Persona3, 25 años
Persona 1:  Persona1, 18 años
Evento 1:  Descripcion uno
Fecha: 18/4/2023
Persona 4:  Persona4, 30 años
Evento 1:  Descripcion uno
Fecha: 19/4/2023
Evento 2:  Descripcion dos
Fecha: 20/4/2023
32>Persona 3:  Persona3, 25 años
33>Persona 4:  Persona4, 30 años
Evento 1:  Descripcion uno
Fecha: 19/4/2023
Evento 2:  Descripcion dos
Fecha: 20/4/2023
34>35>Fin.
